sycl/test-e2e/ESIMD/accessor_global.cpp

```diff
@@ -0,0 +1,126 @@
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip || esimd_emulator
+// UNSUPPORTED: gpu-intel-gen9 && windows
+//
+// RUN: %clangxx -fsycl -fsycl-esimd-force-stateless-mem %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
// RUN: %{build} -fsycl-esimd-force-stateless-mem -o %t.out
// RUN: %{run} %t.out
```

Reply:
Sure. Thank you. Fixed: 38ee4e7fb7920b061730ead26764f3dcdd4f80f5